npm/namespace.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Microsoft. All rights reserved.
+// Package npm use Copyright 2018 Microsoft. All rights reserved.
```

Comment:
What was the linting warning here? The change here seems strange. If there is no simple way to document what the `npm` package does, then better to just leave as is. Same for all similar below.